shared/src/business/entities/Document.js

```diff
@@ -296,12 +305,13 @@ Document.prototype.setAsServed = function(servedParties = null) {
 
 /**
  * generates the filedBy string from parties selected for the document
- * and contact info from the case detail
+and contact info from the case detail
  *
  * @param {object} caseDetail the case detail
+ * @param {boolean} force flag to force filedBy's generation
  */
-Document.prototype.generateFiledBy = function(caseDetail) {
-  if (!this.filedBy) {
+Document.prototype.generateFiledBy = function(caseDetail, force = false) {
+  if (force || !this.filedBy) {
```

Comment:
This was due to a document being changed/edited for Notice of Docket Change (https://github.com/flexion/ef-cms/issues/3247) by a given Docket Clerk (e.g. if `Docket Clerk 1` added a docket entry, and `Docket Clerk 2` modified it, it'd update the `filedBy` on the docket entry (and coversheet). 

Its utilization: https://github.com/ustaxcourt/ef-cms/pull/115/files/9a6e45f9ec51885363b03f2921d2f9a429649d86#diff-496049fd61fff6ed70a8fb82eb81c521R275

Reply:
Since this method is referenced in the constructor, it would get called every time we pass raw data to the entity. We have a check on the function - `!this.filedBy` - to prevent overwriting the property if it has already been computed (the method sets the property on the object). But there are some cases where we _want_ to overwrite it. Adding the force flag would force the overwrite in those cases.